### ISSUE TITLE
Baritone crash with Replay Mod (this.a)

### ISSUE THREAD

**Issue by comand100vip** [cabaletta/baritone#3343]
## Some information
Operating system: `Windows 10 Pro`
Java version: `17.0.2 2022-01-18 LTS`
Minecraft version: `1.18.2`
Baritone version: `1.8.3 standalone fabric`
Fabric Loader version: `0.13.3`
Fabric API version: `0.47.10`
Other mods (if used): 
```
modmenu-3.1.0
replaymod-1.18.2-2.6.4
sodium-fabric-mc1.18.2-0.4.1+replaymod.rev.23e83a1
sodium-extra-mc1.18.2-0.4.2
```

## Exception, error or logs
[lastest.log](https://gist.github.com/comand100vip/9247bb6940233e9b1022b6a5f20b8b41)
[crash report](https://gist.github.com/comand100vip/cc3a62f32b31c5c99104aeaceb6a57d0)

## How to reproduce
```
1. Install all mods in list above.
2. Record any world with it (I recorded it on server)
3. Exit world and go to replay's.
4. Try to load any replay.
```

## Modified settings
None

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x] I have included logs, exceptions and / or steps to reproduce the issue.
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by wagyourtail** [comment]
known about and fixed for a future version as of #3255

**Comment by comand100vip** [comment]
So it is fixed but not yet released?
